tests/NewCypress/cypress/e2e/regression/pagination.cy.ts

```diff
@@ -0,0 +1,82 @@
+import paginationPage from "../../pages/paginationPage";
+
+describe('Pagination Tests', () => {
+
+
+    beforeEach(() => {
+        cy.login()
+        cy.visit('/search?keywords=west')
+    });
+
+    it('Should display multiple pagination buttons', () => {
+        paginationPage.getTotalPaginationButtons().should('be.greaterThan', 1);
+    });
+
+    it('Should navigate to a specific page when a page number is clicked', () => {
+        const targetPage = 3;
+
+        paginationPage
+            .getResults()
+            .then(initialResults => {
+                const initialFirstResultText = initialResults
```

Comment:
This variable name is a little tough to understand, can we change it to something clearer? Maybe `firstPageFirstResult`?

Reply:
Now updated - also changed some similar ones to be a bit clearer.